hack/release.yaml

```diff
@@ -1,6 +1,6 @@
 NetworkOperator:
   image: network-operator
-  repository: nvcr.io/nvidia/cloud-native
+  repository: nvcr.io/nvstaging/mellanox/
```

Comment:
Hey @e0ne, looks like this change breaks helm installation, since image name is getting double` //` after repositery name
```
k -n nvidia-network-operator get pods network-operator-keep-ncp-tcg8h -oyaml | grep image:
    image: nvcr.io/nvstaging/mellanox//network-operator:v24.10.0
```

Reply:
fixed